CMakeLists.txt

```diff
@@ -25,12 +25,17 @@ if(APPLE)
 
   # iOS build options
   option(BUILD_IOS           "Build for iOS"               NO)
+  option(FORCE_RESET_OSX_DEPLOYMENT_TARGET "Clear the OSX Deployment Target Set" ON)
```

Comment:
Value should be YES, so by default current behavior prevails

Reply:
Yes, the value ON is equivalent to YES. You can either use ON/OFF or you can use YES/NO.

Reply:
The project standard is to use YES, please use that.

Reply:
I have update that to YES.